src/nuclide.cpp

```diff
@@ -683,6 +692,24 @@ void Nuclide::calculate_xs(int i_sab, int i_log_union, double sab_frac, Particle
       micro.nu_fission = 0.0;
     }
 
+    if (settings::frequency_method_on) {
+      if (fissionable_) {
+        micro.prompt_nu_fission = (1.0 - f)*(xs(i_grid, XS_FISSION) *
+	  	nu(grid.energy[i_grid], EmissionMode::prompt))
+		+ f*(xs(i_grid + 1, XS_FISSION) * 
+		nu(grid.energy[i_grid + 1], EmissionMode::prompt));
+        for (int d = 0; d < n_precursor_; ++d) {
+	  micro.delayed_nu_fission[d] = (1.0 - f)*(xs(i_grid, XS_FISSION) * 
+		nu(grid.energy[i_grid], EmissionMode::delayed, d+1))
+		+ f*(xs(i_grid + 1, XS_FISSION) *
+		nu(grid.energy[i_grid + 1], EmissionMode::delayed, d+1));
+        }
+      } else {
+	micro.prompt_nu_fission = 0.0;
+	micro.delayed_nu_fission.fill(0.0);
+      }
```

Comment:
Why not evaluate nu at the interpolated energy and multiply by `micro.fission` rather than interpolating the product of nu and fission? Also, the indentation here looks inconsistent.

Reply:
The interpolation of a product is not equal to the interpolation of the parts of that product. I think it would introduce more error to interpolate them separately.
Indentation: sorry, it looks fine in my editor and on my repository in GitHub, but I agree it is wonky here in the PR. Not sure why.

Reply:
> The interpolation of a product is not equal to the interpolation of the parts of that product.

Understood, but it's actually not clear which will be correct. Also, sometimes `nu` is given as a polynomial function rather than a tabulated function, in which case it doesn't make sense to interpolate it when you can evaluate it exactly at the energy of interest. Performance wise, you're also better off evaluating nu once rather than twice.

Regarding indentation -- it looks like you have tab characters in here. We uniformly use spaces in the OpenMC code base, so I would encourage you to ensure that your editor converts tabs to spaces when working on OpenMC.

Reply:
I may have misunderstood your first comment. I'm on board with what you're proposing, and it's in the new commit. I also made sure spaces are being used. So hopefully that takes care of the indentations.

Reply:
I may have misunderstood your first comment. I'm on board with what you're proposing, and it's in the new commit. I also made sure spaces are being used. So hopefully that takes care of the indentations.